conf/mpcdf.config

```diff
@@ -61,7 +61,7 @@ profiles {
        
         params {
             config_profile_description = 'MPCDF raven profile (unofficially) provided by nf-core/configs.'
-            max_memory = 368.GB
+            max_memory = 2000000.MB
```

Comment:
Maybe use GB as units and make it easier to read that way ;-)

Reply:
Unfortunately nextflow seems to interpret/calculate GB differently, and ends up exceeding the strict MB limit allowed on this cluster...

Reply:
Future reference: Likely the Terabyte vs Tebibyte discussion https://www.dataunitconverter.com/terabyte-to-tebibyte/1